p/themes/Origine/origine.css

```diff
@@ -2,10 +2,19 @@
 
 /*=== FONTS */
 @font-face {
-	font-family: "OpenSans";
-	src: url("../fonts/openSans.woff") format("woff");
+  font-family: 'Open Sans';
```

Comment:
Please change line 22 accordingly (mind the space) https://github.com/ASMfreaK/FreshRSS/blob/dev/p/themes/Origine/origine.css#L22